.github/actions/conventional-commits/action.yml

```diff
@@ -1,20 +1,78 @@
 name: Conventional Commits Checking
 description: "Validates that a pull request title is compatible with Conventional Commits."
+inputs:
+  enforce_identifier_prefixes:
+    description: "List of JIRA (or similar) project identifiers to enforce. eg: `feat(FOO-123):...` or `feat(BAR-123):...` then pass in FOO\nBAR"
```

Comment:
I will add that this description is kind of confusing. I'm not really sure what it means.

Reply:
Can you provide an example of what format you are expecting here. Like is it a string of characters that you are allowing or is it a regexp or something else.

Reply:
https://github.com/chanzuckerberg/github-actions/pull/90/files#diff-e27c2676c8ee63a108d6d91680cbeca9fdce68ff977cc4914b149f7e40b065f0R16

Reply:
ya I guess that example doesn't help me understand the newline character mentioned in this description